exampleCourse/questions/examplesPanels/info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "uuid": "81a3e61d-9ecc-457c-9dac-db5b6b451d3c",
+    "title": "Show or hide text in specific panels",
```

Comment:
I think examples normally have the name of the element in the title? e.g.

`Element action via pl-element`

`Real and complex numeric input via pl-number-input`

Reply:
I tried this initially, but we discuss four elements and the title got crazy long with everything listed 😄

I didn’t want to only discuss `pl-hide-in-element` because I wanted to show the list of alternatives.